spotifyapi/models/__init__.py

```diff
@@ -23,3 +25,5 @@
 from .time_interval import TimeInterval
 from .token import Token
 from .track import Track
+from .user import User
+
```

Comment:
Extra new line here, `black` should've caught that on the commit, I need to look into that. For now, run `black` on all the files to resolve issues like this